### ISSUE TITLE
[Turbopack]  Parsing leaflet css source code failed

### ISSUE THREAD

**Issue by sinhong2011** [vercel/turbo#2451]
### What version of Turbopack are you using?

Next 13.0.0

### What package manager are you using / does the bug impact?

pnpm

### What operating system are you using?

Mac

### Describe the Bug

```
error - [parse] [project-with-next]/node_modules/.pnpm/leaflet@1.8.0/node_modules/leaflet/dist/leaflet.css  [project-with-next]/node_modules/.pnpm/leaflet@1.8.0/node_modules/leaflet/dist/leaflet.css:552:32  Parsing css source code failed
     548   	width: 24px;
     549   	margin: 0 auto;
     550   
     551   	-ms-filter: "progid:DXImageTransform.Microsoft.Matrix(M11=0.70710678, M12=0.70710678, M21=-0.70710678, M22=0.70710678)";
     552 > 	filter: progid:DXImageTransform.Microsoft.Matrix(M11=0.70710678, M12=0.70710678, M21=-0.70710678, M22=0.70710678);
     553   	}
     554   
     555   .leaflet-oldie .leaflet-control-zoom,
     556   .leaflet-oldie .leaflet-control-layers,
  
  Expected Declaration value
```

### Expected Behavior

parse leaflet CSS success

### To Reproduce

`pnpm i && pnpm env-cmd -e local next dev --turbo --show-all`

### Reproduction Repo

https://github.com/sinhong2011/coincart-react/tree/develop

**Comment by Skilles** [comment]
Can confirm I'm getting the same issue trying to use syncfusion components. Fails parsing any css that contains a line such as this:
`filter: alpha(opacity=0);`
Seems like the "filter" css property is not properly supported.

**Comment by walmat** [comment]
+1 experiencing this still

**Comment by iSuslov** [comment]
I'm getting similar issue with vidstack:

```
 error - [parse]
   node_modules/vidstack/styles/defaults.css
     node_modules/vidstack/styles/defaults.css:639:28  Parsing css source code failed
        635 |   writing-mode: horizontal-tb;
        636 |   top: calc(
          637 |     var(
        638 |       --region-top,
            +                             v
        639 +       var(--overlay-height) * var(--region-viewport-anchor-y) / 100 - var(--region-height) *
            +                             ^
        640 |         var(--region-anchor-y) / 100
        641 |     )
        642 |   );
        643 |   left: var(

     Expected Declaration value
   node_modules/vidstack/styles/ui/captions.css
     node_modules/vidstack/styles/ui/captions.css:111:28  Parsing css source code failed
        107 |   writing-mode: horizontal-tb;
        108 |   top: calc(
          109 |     var(
        110 |       --region-top,
            +                             v
        111 +       var(--overlay-height) * var(--region-viewport-anchor-y) / 100 - var(--region-height) *
            +                             ^
        112 |         var(--region-anchor-y) / 100
        113 |     )
        114 |   );
        115 |   left: var(

     Expected Declaration value
```

**Comment by lutfi-haslab** [comment]
Same here:
```
Server Error

Error: error - node_modules/leaflet/dist/leaflet.css:551:15  Parsing css source code failed

  549 | 
  550 | 	-ms-filter: "progid:DXImageTransform.Microsoft.Matrix(M11=0.70710678, M12=0.70710678, M21=-0.70710678, M22=0.70710678)";
> 551 | 	filter: progid:DXImageTransform.Microsoft.Matrix(M11=0.70710678, M12=0.70710678, M21=-0.70710678, M22=0.70710678);
      | 	              ^
  552 | 	}
  553 | 
  554 | .leaflet-oldie .leaflet-control-zoom,
Expected Declaration value
```
using next dev --turbo

**Comment by chrishoermann** [comment]
same here

**Comment by vaupunkt** [comment]
+1 yes same here

**Comment by ryanrosenberg** [comment]
also running into this issue. is this something that needs to be fixed on the `react-leaflet` side?

**Comment by yannicschroeer** [comment]
The error occurs in the `leaflet` base library, not even `react-leaflet`.

This is the issue: 

**leaflet/dist/leaflet.css [655:661]**
```css
.leaflet-oldie .leaflet-popup-tip {
	width: 24px;
	margin: 0 auto;

	-ms-filter: "progid:DXImageTransform.Microsoft.Matrix(M11=0.70710678, M12=0.70710678, M21=-0.70710678, M22=0.70710678)";
	filter: progid:DXImageTransform.Microsoft.Matrix(M11=0.70710678, M12=0.70710678, M21=-0.70710678, M22=0.70710678);
}
```

But I can not find the responsible lines in the source https://github.com/Leaflet/Leaflet/blob/main/dist/leaflet.css

**Comment by vaupunkt** [comment]
I made a local Copy of the CSS and removed the three lines and now it works perfectly 😮‍💨🤷‍♂️

**Comment by arashgl** [comment]
same problem here

**Comment by jovermier** [comment]
It looks like it was added to the `dist/leaflet.css` with this commit:
https://github.com/Leaflet/Leaflet/commit/45181ce40b4c3991b5d58ce75e54a76b97ab3945#diff-9fa3a4eecc1d108f0f3bc430fa16b55d7998dd8630cb95b36fd817ed4b367155R485

Then removed in this commit https://github.com/Leaflet/Leaflet/commit/2daebbb4390b8e52c0b8c7bfc2074fe577f8bf73

It appears `dist/leaflet.css` current version is no longer being bundled and instead an older version is being used. I diffed the `dist/leaflet.css` file changes until I found similar contents. The issue happens between commits:
https://github.com/Leaflet/Leaflet/commit/0fa492c239e10ab21d688f7ea1afe9f6fefe6505 and https://github.com/Leaflet/Leaflet/commit/0fa492c239e10ab21d688f7ea1afe9f6fefe6505

What is weird is there is never a commit that has an exact match of the contents of the `dist/leaflet.css` file.
